File: problemas/1000/1043.dart
import 'dart:io';

void main() {
  final entrada = stdin.readLineSync().split(' ');
  final a = double.parse(entrada[0]);
  final b = double.parse(entrada[1]);
  final c = double.parse(entrada[2]);

  if ((b - c).abs() < a && (a - c).abs() < b && (a - b).abs() < c &&
    a < b + c && b < a + c && c < a + b) {
      stdout.writeln('Perimetro = ${(a + b + c).toStringAsFixed(1)}');
    } else {
      stdout.writeln('Area = ${((a + b) * c / 2).toStringAsFixed(1)}');
    }
}

File: problemas/1000/1043.py
(a, b, c) = map(float, input().split(' '))

if abs(b - c) < a < (b + c) and abs(a - c) < b < (a + c) and abs(a - b) < c < (a + b):
    print('Perimetro = {:.1f}'.format(a + b + c))
else:
    print('Area = {:.1f}'.format((a + b) * c / 2))
